packages/block/README.md

```diff
@@ -60,11 +60,12 @@ Properties of a `Block` or `BlockHeader` object are frozen with `Object.freeze()
 API Usage Example:
 
 ```ts
+// ./examples/1559.ts#L43-L47
+
```

Comment:
@scorbajio Thanks for the different doc updates! 🙂 

One note on the per-line example references: on the one hand I find this elegant. On the other hand I fear a bit that we wil rather overlook this mid-term and it will relatively likely happen that we will do some example updates (and e.g. add a few lines to some examples), globally rebuild the docs example integration and then overlook that the lines are not matching anymore.

Maybe let's therefore be a bit careful with this.